host/xtest/xtest_1000.c

```diff
@@ -577,6 +741,23 @@ static void xtest_tee_test_1008(ADBG_Case_t *c)
 		}
 	}
 	Do_ADBG_EndSubCase(c, "Create session fail");
+
+	Do_ADBG_BeginSubCase(c, "Load fake uuid TA");
+	load_fake_ta(c);
+	Do_ADBG_EndSubCase(c, "Load fake uuid TA");
+
+	Do_ADBG_BeginSubCase(c, "Load corrupt TA");
+	ADBG_EXPECT_TRUE(c, load_corrupt_ta(c,  0, 1)); /* magic */
```

Comment:
Except that the struct isn't available here. Are you proposing that we should export `signed_hdr.h` in `host_includes`?